run/websockets/public/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "browser": true
+  },
+  "rules": {
+    "no-undef": "off"
```

Comment:
@grayside @JustinBeckwith is this rule-ignore OK here?

(I noticed you had issues with this in other folders.)

Reply:
This is needed to ignore `document`, `mdc` (material design), and `$` (jquery)